detox/ios/Detox/DetoxInstrumentsManager.m

```diff
@@ -234,10 +250,10 @@ - (instancetype)init
 	return self;
 }
 
-- (id)_configForDetoxRecordingWithURL:(NSURL*)URL andSamplingInterval:(NSTimeInterval)samplingInterval
+- (DetoxInstrumentsConfiguration *)configurationFromProps:(NSDictionary *)props
```

Comment:
Do you mind changing this to `configurationWithDictionary:`

Reply:
Thats fine. I'll change.